base/sysinfo.jl

```diff
@@ -32,6 +32,7 @@ export BINDIR,
        isunix,
        iswindows,
        isjsvm,
+       show,
```

Comment:
```suggestion
```
You don't need to export `show` — it is not a new *function* in this module, it is a new *method* added to an existing `Base.show` function.